commit e929c07b1e7a8c4b52e5efa11cc664f4f41c0a78 (HEAD -> master, refs/changes/97/4497/2)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-08-02T15:32:34+00:00 (1 year, 2 months ago)
    
    joyent/node-assert-plus#32 package.json repository does not point to Joyent
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/package.json b/package.json
index 40d6a5c..9cb0e2d 100644
--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/mcavage/node-assert-plus.git"
+    "url": "https://github.com/joyent/node-assert-plus.git"
   },
   "engines": {
     "node": ">=0.8"
